client/components/social-buttons/facebook.js

```diff
@@ -24,6 +26,7 @@ class FacebookLoginButton extends Component {
 		status: false,
 		xfbml: true,
 		scope: 'public_profile,email',
+		onClick: noop,
```

Comment:
Interesting. I have never thought about that but this `noop` approach is quite slick.